src/plugins/discourse/createGraph.js

```diff
@@ -173,19 +199,26 @@ class _GraphCreator {
       this.graph.addEdge(authorsTopicEdge(serverUrl, topic));
     }
 
+    const postDescriptions: Map<PostId, string> = new Map();
```

Comment:
Nit: inconsistent with `topicIdToTitle` which sets the map as a class property.

Reply:
Refactored for consistency.